pypdf/_doc_common.py

```diff
@@ -998,16 +1004,15 @@ def _build_outline_item(self, node: DictionaryObject) -> Optional[Destination]:
     def pages(self) -> List[PageObject]:
         """
         Property that emulates a list of :class:`PageObject<pypdf._page.PageObject>`.
-        this property allows to get a page or a range of pages.
-
-        For PdfWriter Only:
-        Provides the capability to remove a page/range of page from the list
-        (using the del operator).
-        Note: only the page entry is removed. As the objects beneath can be used
-        elsewhere.
-        A solution to completely remove them - if they are not used anywhere -
-        is to write to a buffer/temporary file and to load it into a new PdfWriter
-        object afterwards.
+        This property allows to get a page or a range of pages.
+
+        .. note::
```

Comment:
This is formatted incorrectly and thus ignored: https://pypdf--2711.org.readthedocs.build/en/2711/modules/PdfWriter.html#pypdf.PdfWriter.pages I recommend you to test the correct rendering locally. The last time we did this looked like this: https://github.com/py-pdf/pypdf/pull/2695/files#diff-2376a2cb969cf3bd675e8f2a9623f3379907210fc6532ffd1e84b9cadae9b37dR98-R100

Reply:
Changed.

Reply:
This still looks broken. I am not sure what the correct command is here, but apparently now we have some code rendering.

The alternative approach to try here might be something like this:

```
.. note::
   The text
```

Reply:
Will read about how this works and work out best way.